src/interfaces/index.ts

```diff
@@ -0,0 +1,20 @@
+export interface INetwork {
+  name: string;
+  rpcUrl: string;
```

Comment:
I would remove the rpcUrl from here. This value may be sensitive and diferent for diferent enviroments

Reply:
import PriceOracle from './priceOracle/priceOracle';

Reply:
I agree, I will add this rpc information in the be by environment variable

Reply:
nice